storage/innobase/xtrabackup/src/xtrabackup.cc

```diff
@@ -2154,6 +2157,20 @@ xb_write_delta_metadata(const char *filename, const xb_delta_info_t *info)
 	return(ret);
 }
 
+static my_bool
+xtrabackup_write_info(const char *filepath)
+{
+	char *xtrabackup_info_data = get_xtrabackup_info(mysql_connection);
+	if (!xtrabackup_info_data) {
+		return FALSE;
+	}
+
+	bool result = write_to_file(filepath, xtrabackup_info_data);
```

Comment:
write_to_file returns my_bool, but result is assigned to my_bool. Please either change write_to_file or the type of "result" variable.